daft/connect/__init__.py

```diff
@@ -0,0 +1,115 @@
+"""Daft Connect provides a simple way to start a Spark server and connect to it from a client.
+
+Additionally, Daft Connect provides a Ray remote class that wraps the the daft.daft.connect_start function.
+
+Usage:
+```py
+from daft.daft import connect_start
```

Comment:
nit: should we explicitly name this `start_spark_connect_server()`?

Also perhaps namespace this outside of `daft.daft` -- potentially either `daft.spark` or just `daft` itself

Reply:
oh yeah this is under `daft.connect` will update docstring.